CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
                 and CFLAGS
  * [`changed`]  Only one example with either `hw_i2c` or `sw_i2c` is built,
                 depending on `CONFIG_I2C_TYPE`. Defaults to `hw_i2c`.
+ * [`fixed`]  Measurement duration could take at most 15.5ms (datasheet) but we
+              only waited 15ms
+ * [`added`]  Add `STS3X_MEASUREMENT_DURATION_USEC` to header
```

Comment:
These changes have nothing to do with the test pipeline, have they?

Reply:
Kinda: They have to do with the tests ran during the pipeline.